spec/amber/controller/respond_with_spec.cr

```diff
@@ -30,6 +38,14 @@ module Amber::Controller
         context.response.status_code.should eq 200
       end
 
+      it "responds with json having */* at end" do
+        expected_result = %({"type":"json","name":"Amberator"})
+        context.request.headers["Accept"] = "application/json;*/*"
```

Comment:
This looks good. Can we get one more test where it asks for an unsupported format, then `*/*`, perhaps `"application/ndjson;*/*;`

Reply:
Just a moment

Reply:
And it is failing

Reply:
hmm, that sounds like it is on the client side. @amberframework/core-team - any idea if this would be in amber or in crystal's HTTP client?

Reply:
@marksiemers It was my fault, because separator between formats is `,`, not the `;`. I updated it with https://github.com/amberframework/amber/pull/586/commits/8d57895880035ada6ae8dccb4f81fc92f91e971a